src/Components/ServiceScene/Breakdowns/LabelValuesBreakdownScene.tsx

```diff
@@ -117,53 +118,73 @@ export class LabelValuesBreakdownScene extends SceneObjectBase<LabelValueBreakdo
         }
       }
     }
+  }
+
+  private setErrorStates(newState: SceneDataState) {
+    // If panels have errors
+    if (newState?.data?.errors && newState.data?.state !== LoadingState.Done) {
```

Comment:
Should we consider when LoadingState is streaming?

Reply:
Yes, but isn't that what's its doing by checking that the loading state is not done?
`newState.data?.state !== LoadingState.Done`
_____________________^ :P 

This hasn't been tested with sharding very well tho, I'll take a look before closing this out.

Reply:
Yeah, that's my point. Some done things but with errors will have an Error state, or other times Error but state is still Streaming, which can cause unexpected behaviors.

Reply:
Took me a bit to get it to fail, but yes, we are showing the error in each panel and as a toast when thrown, but only when sharding is done retrying.

<img width="1719" alt="image" src="https://github.com/user-attachments/assets/db658d48-0b82-48aa-97ee-5e6e7c97a299">